content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -186,6 +185,38 @@ func failableFunction() (any, error) {
 }
 {{</code-block>}}
 
+#### Prevent instrumentation of some code
+
+The `//orchestrion:ignore` directive can be used to prevent `orchestrion` from performing _any_ modification on the annotated code.
```

Comment:
```suggestion
You can use the `//orchestrion:ignore` directive to prevent `orchestrion` from performing _any_ modification on the annotated code.
```